news.html

```diff
@@ -8,13 +8,21 @@
 
     <div class="container community-wrapper">
 
-        <div class="col-sm-8 wrapper">
+        <div class="col-sm-12 wrapper">
```

Comment:
Instead of making the section wider, I think we should just make this particular bit of text shorter